neofetch

```diff
@@ -3534,6 +3540,11 @@ get_distro_colors() {
             ascii_file="porteus"
         ;;
 
+        "Proxmox VE"*)
+            set_colors 7 202
```

Comment:
Are we authorized to use 24 bits based colors here? The color chosen here is not really the same orange color as Proxmox.
https://en.wikipedia.org/wiki/ANSI_escape_code#24-bit
If not, maybe we could add a note to the README or the wiki in order to help newscomers?